SPECS/kernel-hyperv/config

```diff
@@ -4104,10 +4104,11 @@ CONFIG_PKCS7_MESSAGE_PARSER=y
 #
 CONFIG_MODULE_SIG_KEY="certs/signing_key.pem"
 CONFIG_SYSTEM_TRUSTED_KEYRING=y
-CONFIG_SYSTEM_TRUSTED_KEYS=""
+CONFIG_SYSTEM_TRUSTED_KEYS="certs/mariner.pem"
```

Comment:
Do you know, if there's a way to allow appending/modifying the trusted keys in the future? I'm thinking about a scenario where we start signing with another key, while the old one is still valid. I'm also curious, if we have a strategy for revoking this key, even though I hope this scenario never happens.

Reply:
The changes also enable the .blacklist keyring so we can revoke keys if needed.